### ISSUE TITLE
Remove custom Composer install path for drupal-custom-module packages

### ISSUE THREAD

**Issue by richardbporter** [uiowa/uiowa#2630]
### Problem
Currently, we install drupal-custom-modules to the custom path `docroot/modules/uiowa` because they are not contrib but can't go in custom otherwise they'd be tracked in VCS. However, this is a really odd location and could cause confusion if we use a non-UI custom module in the future.

### Possible solutions
1. ~~Remove the custom install path and treat anything not in this repository as contributed. This would move uiowa_auth to `docroot/modules/contrib` which would require a cache rebuild.~~

2. Copy/paste uiowa_auth into this repository in `docroot/modules/custom` and ~~then make use of GitHub release file uploads for extracting and sharing that module, _if necessary_~~.  Update the composer.json file to remove the custom install path to `docroot/modules/uiowa`.

~~This is blocked by #1907 and #1713 which are blocked by #2629.~~

**Comment by richardbporter** [comment]
After struggling with maintaining a test environment for uiowa_auth many times, including recently, I'm firmly in favor of option 2. Getting a custom module and dependencies in a full Drupal install is tricky. We should just piggy-back off the full Drupal install provided by this repo and BLT. I think we could maybe use this method for providing other assets extracted from this repo.

**Comment by richardbporter** [comment]
https://docs.travis-ci.com/user/deployment/releases/#uploading-multiple-files

**Comment by pyrello** [comment]
@richardbporter Since 2 is not strictly necessary for the post HR clean-up, I'm inclined to just make the changes in 1 above for now and leave 2 as future work.

**Comment by richardbporter** [comment]
Could we do 2 but without the release uploading? We have no use case for sharing this module with anyone at this point.

**Comment by pyrello** [comment]
If we are not sharing the module, then I think that makes the most sense. It would certainly make the whole process easier.

**Comment by richardbporter** [comment]
Proposed solution: option 2 without any release uploading.
